extras/gomemif/memif/interface.go

```diff
@@ -367,6 +368,22 @@ func eventFd() (efd int, err error) {
 	return int(u_efd), nil
 }
 
+// The commented function is advance preparation for potential replacement of EpollWait syscall
+// func EpollPWait(epfd int, events []EpollEvent, msec int, sigmask sigmask) (n int, err error) {
```

Comment:
We can keep this function in preparation, but we should probably not comment it out

Reply:
Fixed